packages/components/src/internal/components/forms/BulkUpdateForm.tsx

```diff
@@ -209,7 +215,9 @@ export class BulkUpdateForm extends PureComponent<Props, State> {
     };
 
     renderBulkUpdateHeader() {
-        const { header } = this.props;
+        const { header, showHeader } = this.props;
+        if (!showHeader) return null;
```

Comment:
Just want to verify that we're adding this prop, and doing this check, because there are cases where we aren't passing a header, and we want the default rendering below. If not, them maybe we could just do `if (!header) return null;`.

Reply:
Nope, good catch there—these two props can definitely be just one prop!